platform-coop-toolkit/coop-map-directory-index
# feat: add Role.icon

## Description

Adds a CharField to the Role model for an icon, which can be used to reference a Pinecone icon ID for each role. Similar to `SocialMedia.font_awesome`.

## Steps to test

1. Check out branch.
2. Run migrations for `accounts`.
3. Visit Roles in admin.

**Expected behavior:** A new text field appears for "Icon" on each role.

## Additional information

Not applicable.

## Related issues

Blocks #28.
